External/FEXCore/include/FEXCore/IR/IntrusiveIRList.h

```diff
@@ -170,6 +170,14 @@ class IRListView final {
     }
   }
 
+  void *operator new(size_t size) {
```

Comment:
Did you end up finding a need to keep these?

Reply:
Ah yes, I see now. Since these overloaded operators all function the same, how do you feel about defining a helper struct with them that uses inherit from?

For example:
```cpp
struct FexAllocOperators {
    void *operator new(size_t size) {
        return FEXCore::Allocator::malloc(size);
    }

  void operator delete(void *ptr) {
    return FEXCore::Allocator::free(ptr);
  }
};

struct IRListView : FexAllocOperators {
    // ...
};
```

Reply:
Switched over to inheritance.